server.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 // smtp network server
 
+// var log = require('why-is-node-running');
```

Comment:
Says the guy that use to be adamantly opposed to such a thing. ;-)  Welcome to the dark side. ;-)